### PR TITLE
VIH-8966 - Add Admin Web to SDS

### PR DESCRIPTION
**Pull request by v1ferrarij**
### JIRA link (if applicable) ###

[VIH-8966](https://github.com/hmcts/vh-admin-web/tree/VIH-8966)

### Change description ###

VIH-8966 - Add Admin Web to SDS

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: .github/workflows/close-pr.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - closed
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  ado_org: 'hmcts'
+  ado_project: 'Video Hearings'
+  ado_pipeline: 'hmcts.vh-admin-web.sds.pr-release'
```
Comment (chris-pateman) [inline .github/workflows/close-pr.yml]: This is correct, well done :P

File: AdminWebsite/AdminWebsite/Dockerfile [hunk 2]
```diff
@@ -26,4 +26,6 @@ FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
 COPY --from=client /app/dist /app/ClientApp/dist
+RUN apt-get update && \
```
Comment (chris-pateman) [inline AdminWebsite/AdminWebsite/Dockerfile:29]: Is this something needed for all apps?

File: azure-pipelines.sds.pr-release.yml [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+        displayName: "Helm CI"
+        steps:
+          - checkout: self
+
+          - template: templates\Containerisation\helm\helm-lint.yaml@azTemplates
+            parameters:
+              chartPath: ${{ variables.chartPath }}
+              chartValuesPath: ${{ variables.chartPath }}
+
+      # - job: UnitTests
```
Comment (chris-pateman) [inline azure-pipelines.sds.pr-release.yml:42]: does this project not have unit tests?

File: azure-pipelines.sds.pr-release.yml [hunk 4]
```diff
@@ -0,0 +1,489 @@
...
+
+      # - job: package_nuget
+      #   displayName: "Package NuGet Packages"
+      #   dependsOn: Dock
+      #   steps:
+      #     - template: templates\dotnet\package-nuget-packages.yml@azTemplates
+      #       parameters:
+      #         nugetProjectPath: "TestApi/${{ variables.nuget_package_name }}"
+      #         nugetConfigFile: "${{ variables.app_path }}/nuget.config" # Temp - Change To SDS Feed When Possible.
+      #         gitVersionConfigPath: ${{ variables.git_version_config_path }}
```
Comment (chris-pateman) [inline azure-pipelines.sds.pr-release.yml:155]: c

File: azure-pipelines.sds.pr-release.yml [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+  #           parameters:
+  #             sqlServerResourceGroup: ${{ variables.vhResourceGroup }}
+  #             sqlServerName: ${{ variables.vhSQLServerName }}
+  #             databaseName: ${{ variables.TestApiDbName }}
+  #             azureSubscription: ${{ variables.subscriptionName }}
+  #             sqlScriptLocation: "$(Pipeline.Workspace)/${{ variables.efContextName }}-$(Build.BuildId)/${{ variables.efContextName }}.sql"
+  #             kvfirewallRequired: false
+  #             kvName: ${{ variables.vhKeyVault }}
+  #             kvSqlPasswordSecret: ${{ variables.vhSqlPasswordSecret }}
+  #             kvSqlUsernameSecret: ${{ variables.vhSqlUsernameSecret }}
```
Comment (chris-pateman) [inline azure-pipelines.sds.pr-release.yml:199]: delete if not needed

File: azure-pipelines.sds.pr-release.yml [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+  #     - template: variables/shared.yaml
+  #       parameters:
+  #         env: ${{ variables.env }}
+  #   jobs: 
+  #     - job: Publish_Nuget
+  #       displayName: Publish NuGet Packages
+  #       steps:
+  #         - template: templates\dotnet\push-nuget-packages.yml@azTemplates
+  #           parameters:
+  #             vstsFeed: "${{ variables.nuget_org_name }}/${{ variables.nuget_feed_name }}"
```
Comment (chris-pateman) [inline azure-pipelines.sds.pr-release.yml:388]: delete if not needed

File: azure-pipelines.sds.pr-release.yml [hunk 7]
```diff
@@ -0,0 +1,489 @@
...
+  #           parameters:
+  #             sqlServerResourceGroup: ${{ variables.vhResourceGroup }}
+  #             sqlServerName: ${{ variables.vhSQLServerName }}
+  #             databaseName: ${{ variables.testApiDbName }}
+  #             azureSubscription: ${{ variables.subscriptionName }}
+  #             sqlScriptLocation: "$(Pipeline.Workspace)/${{ variables.efContextName }}-$(Build.BuildId)/${{ variables.efContextName }}.sql"
+  #             kvfirewallRequired: false
+  #             kvName: ${{ variables.vhKeyVault }}
+  #             kvSqlPasswordSecret: ${{ variables.vhSqlPasswordSecret }}
+  #             kvSqlUsernameSecret: ${{ variables.vhSqlUsernameSecret }}
```
Comment (chris-pateman) [inline azure-pipelines.sds.pr-release.yml:433]: delete if not needed

File: azure-pipelines.sds.release.yml [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+  #             kvSqlUsernameSecret: ${{ variables.vhSqlUsernameSecret }}
+
+  #         - ${{ if or( eq(env, 'Staging'), eq(env, 'Prod') ) }}: 
+  #           - template: templates/Azure/Common/unlock-lock.yaml@azTemplates
+  #             parameters:
+  #               addOrDelete: add
+  #               lockName: "${{ variables.env }}-lock"
+  #               resourceGroup: ${{ variables.vhResourceGroup }}
+  #               azureSubscription: ${{ variables.subscriptionName }}
+  #               lockType: CanNotDelete
```
Comment (chris-pateman) [inline azure-pipelines.sds.release.yml:143]: delete if not needed

File: azure-pipelines.sds.release.yml [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+
+  - stage: Approve_Helm_Release
+    displayName: 'Approve Helm Release'
+    # ${{ if eq(variables['Build.SourceBranchName'], 'master') }}:
+    #   dependsOn:
+    #     - Run_Entity_Framework_Dev
+    #     - Run_Entity_Framework_Demo
+    #     - Run_Entity_Framework_Staging
+    # ${{ if contains(variables['Build.SourceBranch'], 'refs/heads/release') }}:
+    #   dependsOn: Run_Entity_Framework_Prod
```
Comment (chris-pateman) [inline azure-pipelines.sds.release.yml:157]: delete if not needed

File: azure-pipelines.sds.release.yml [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+  #     - template: variables/shared.yaml
+  #       parameters:
+  #         env: ${{ variables.env }}
+  #   jobs: 
+  #     - job: Publish_Nuget
+  #       displayName: Publish NuGet Packages
+  #       steps:
+  #         - template: templates\dotnet\push-nuget-packages.yml@azTemplates
+  #           parameters:
+  #             vstsFeed: "${{ variables.nuget_org_name }}/${{ variables.nuget_feed_name }}"
```
Comment (chris-pateman) [inline azure-pipelines.sds.release.yml:230]: delete if not needed

File: charts/vh-admin-web/values.yaml [hunk 11]
```diff
@@ -17,41 +17,38 @@ java:
...
+    VHSERVICES__VALIDATEEMAIL: "@hearings.reform.hmcts.net"
     VHSERVICES__VIDEOAPIRESOURCEID: https://vh-video-api.{{ .Values.global.environment }}.platform.hmcts.net
     VHSERVICES__VIDEOAPIURL: https://vh-video-api.{{ .Values.global.environment }}.platform.hmcts.net/
-    VHSERVICES__NOTIFICATIONAPIRESOURCEID: https://vh-notification-api.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__NOTIFICATIONAPIURL: https://vh-notification-api.{{ .Values.global.environment }}.platform.hmcts.net/
-    KINLYCONFIGURATION__CONFERENCEPHONENUMBER: "01234567890"
-    KINLYCONFIGURATION__JOINBYPHONEFROMDATE: ""
-    WOWZACONFIGURATION__STORAGEACCOUNTNAME: vhinfrawowza{{ .Values.global.environment }}
-    WOWZACONFIGURATION__STORAGECONTAINERNAME: recordings
+    VHSERVICES__VIDEOWEBURL: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net/
```
Comment (chris-pateman) [inline charts/vh-admin-web/values.yaml:54]: URLs will need overrides in flux for staging and prod

File: variables/shared.yaml [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+variables:
+
+  - name: nuget_org_name
+    value: 'Video Hearings/vh-packages'
+  - name: nuget_feed_name
+    value: 'vh-packages'
+  - name: git_version_config_path
+    value: $(Build.SourcesDirectory)/sds-git-version-config.yml
+  - name: nuget_package_name
+    value: 'AdminWebsite.Client'
```
Comment (chris-pateman) [inline variables/shared.yaml:18]: don't need feed vars

File: variables/shared.yaml [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+  # - name: efContextName
+  #   value: TestApiDbContext
+  # - name: efProjectName
+  #   value: TestApi.csproj
+  # - name: vhKeyVault
+  #   value: vh-infra-core-${{ parameters.env }}
+  # - name: vhSqlUsernameSecret
+  #   value: "vh-infra-core-${{ parameters.env }}-sql-username"
+  # - name: vhSqlPasswordSecret
+  #   value: "vh-infra-core-${{ parameters.env }}-sql-password"
```
Comment (chris-pateman) [inline variables/shared.yaml:86]: delete if not needed

**Review by chris-pateman — CHANGES_REQUESTED** [review]

File: charts/vh-admin-web/values.yaml (hunk 11 above)
Comment (v1ferrarij) [reply charts/vh-admin-web/values.yaml:54]: already done it.

File: AdminWebsite/AdminWebsite/Dockerfile (hunk 2 above)
Comment (v1ferrarij) [reply AdminWebsite/AdminWebsite/Dockerfile:29]: no, need to delete this actually as was for debugging

File: azure-pipelines.sds.pr-release.yml (hunk 3 above)
Comment (v1ferrarij) [reply azure-pipelines.sds.pr-release.yml]: yeah, they seem to be working now... haha

**Review by chris-pateman — APPROVED** [review]
LGTM

**Pull request merged**